packages/react-query-4/src/QueryClientConsumer.tsx

```diff
@@ -0,0 +1,12 @@
+import { type QueryClient, useQueryClient } from '@tanstack/react-query'
+import type { ReactNode } from 'react'
+
+export function QueryClientConsumer({
+  children,
+  context,
+}: Readonly<{
+  children: (queryClient: QueryClient) => ReactNode
+  context?: React.Context<QueryClient | undefined>
+}>) {
```

Comment:
```suggestion
}: {
  children: (queryClient: QueryClient) => ReactNode
  context?: React.Context<QueryClient | undefined>
}) {
```

Do we need Readonly? I want to know your intention

Reply:
My intention was to prevent the defined `queryClient` or `context` from being reassigned due to user error. TanStack Query has a high dependency on `queryClient`, so I wanted to reduce mistakes by preventing changes.

Upon reflection, I believe this was a hasty approach, so I have reverted the changes to their original state.